com.jetbrains.RustRover.yml

```diff
@@ -0,0 +1,139 @@
+app-id: com.jetbrains.RustRover
+command: rustrover
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  - --allow=devel
+  - --device=all
+  - --env=RUST_ROVER_PROPERTIES=/app/bin/idea.properties
+  - --filesystem=host
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=xdg-run/docker
+  - --filesystem=xdg-run/gnupg
+  - --filesystem=xdg-run/keyring
+  - --filesystem=xdg-run/podman
+  - --share=ipc
+  - --share=network
+  - --socket=gpg-agent
+  - --socket=ssh-auth
+  - --socket=x11
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
needs an exception https://docs.flathub.org/docs/for-app-authors/linter#exceptions https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
updated PR

Reply:
was that permission necessary or not necessary?

Reply:
it was initially in the Jetbrains IDEA community flatpak I used as template for this one -> https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Community/commit/1c15cb47d83d359c195b474bf3969c7a6faa266c#diff-918f291bc83e65d3de5aa74585a328ac4f1b6d7efc9465e4769cfcdb683765e3L20

Reply:
@bbhtt mostly looking around and aligning with other Flathub Jebrains flatpak builds - e.g. this one as well is different, but make sense as RustRover like CLion relies on native code debugging capabilities: https://github.com/flathub/com.jetbrains.CLion/pull/8